staging/src/k8s.io/client-go/examples/dynamic-create-update-delete-deployment/README.md

```diff
@@ -0,0 +1,83 @@
+# Create, Update & Delete Deployment
+
+This example program demonstrates the fundamental operations for managing on
+[Deployment][1] resources, such as `Create`, `List`, `Update` and `Delete` using the `dynamic` package.
+
+You can adopt the source code from this example to write programs that manage
+other types of resources through the Kubernetes API.
+
+## Running this example
+
+Make sure you have a Kubernetes cluster and `kubectl` is configured:
+
+    kubectl get nodes
+
+Compile this example on your workstation:
+
+```
+cd create-update-delete-deployment
```

Comment:
this is still pointing to the typed-client example. I'd suggest having one README for both examples instead of copying it twice